streams/src/main/java/org/apache/kafka/streams/processor/internals/ClientUtils.java

```diff
@@ -90,24 +96,21 @@ public static String getTaskProducerClientId(final String threadClientId, final
         return result;
     }
 
-    public static Map<TopicPartition, ListOffsetsResultInfo> fetchEndOffsetsWithoutTimeout(final Collection<TopicPartition> partitions,
```

Comment:
Removed this as we presumably always want to apply a timeout and do not want `KafkaStreams#allLocalStorePartitionLags` to block indefinitely